presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -54,13 +54,14 @@ Hive does not exactly follow the SQL standard, there are the following
 limitations and differences:
```

Comment:
Please squash commits into single commit and follow https://chris.beams.io/posts/git-commit/ when writting a commit message.